honeybee_vtk/config.py

```diff
@@ -229,3 +250,38 @@ class Config(BaseModel):
         description='List of data-config objects that define the data to be loaded on'
         ' the model.'
     )
+
+
+class DateTimeConfig(BaseModel):
```

Comment:
I'm not sure why do you need this object? Why don't you use Ladybug's DateTime type directly?

Reply:
I don't understand  your comment here. How can you make a Ladybug tools DateTime object in json file?